installing/installing_other/install-rhacs-other.adoc

```diff
@@ -11,7 +11,7 @@ toc::[]
 
 Before you install:
 
-* Understand the xref:../../installing/acs-installation-platforms.adoc#install-platforms-methods_acs-installation-platforms[installation platforms and methods].
+* Understand the xref:../../installing/acs-high-level-overview.adoc#install-platforms-methods[installation methods for different platforms].
```

Comment:
I could not find this xref to ensure that it works!

Reply:
Thanks for catching that, somehow it wasn't included in the topic map!